haskell/src/Monocle/Env.hs

```diff
@@ -25,7 +25,8 @@ data Env = Env
 -- | 'Env' is the global environment
 data AppEnv = AppEnv
   { config :: IO (Bool, Config.Config),
-    aEnv :: Env
+    aEnv :: Env,
+    cRStatus :: IORef [(Config.Index, Bool)]
```

Comment:
I think we need to use a TVar here because `servant` uses async handlers and we don't want to run updateCrawlerMD in parallel.